internal/pkg/overlay/overlay.go

```diff
@@ -505,3 +505,7 @@ func OverlayGetFiles(name string) (files []string, err error) {
 	})
 	return
 }
+
+func Exists(name string) bool {
+	return util.IsDir(OverlaySourceDir(name))
```

Comment:
OverlaySourceDir is undefined, guessing not all changes are pushed.

Reply:
@JasonYangShadow this was actually a problem of rebasing on main and not fully accommodating changes to overlays from upstream. Should be fixed now. Thanks for pointing it out!